src/aioftp/client.py

```diff
@@ -1190,6 +1276,19 @@ async def get_stream(self, *command_args, conn_type="I", offset=0):
             throttles={"_": self.throttle},
             timeout=self.socket_timeout,
         )
+        if self.explicit_tls:
```

Comment:
Maybe this should be a private attribute with meaning of «upgraded to tls» or something like this